src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
minor glitch here

Reply:
Weird, first time I'm seeing this glitch. Thanks!